app/api/routers/attributes.py

```diff
@@ -12,10 +12,19 @@ async def get_term_labels_for_vocab(
     data_element_URI: DataElementURI, request: Request
 ):
     """When a GET request is sent, return a dict containing the name, namespace info, and all term ID-label mappings for the vocabulary of the specified variable."""
-    if data_element_URI is DataElementURI.assessment:
-        return await crud.get_term_labels_for_cogatlas(
-            term_labels_path=request.app.state.cogatlas_term_lookup_path
+    if data_element_URI == DataElementURI.assessment:
+        return await crud.get_term_labels_for_vocab(
+            term_labels_path=request.app.state.cogatlas_term_lookup_path,
+            vocabulary_name="Cognitive Atlas Tasks",
+            namespace_prefix="cogatlas",
         )
+    else:
```

Comment:
I think this `else` is not needed here.

It's a (relatively) new language feature, but you could also take a look at https://peps.python.org/pep-0634/#the-match-statement